### ISSUE TITLE
Synchronization blip

### ISSUE THREAD

**Issue by supermensa** [standardnotes/desktop#672]
I experience this a lot, even though I'm on fast networks (goes for both desktop app and mobile):

When creating or updating a file, the app temporarily fails to synchronize and then creates a duplicate version of said file that differs from the one synchronized. Sometimes this happens as I'm in the middle of writing a sentence. It seems incredibly fast to create duplicate a file that differs from the non synchronized version in a matter of seconds.
Is it possible to either delay this a little or make this a variable that you could tweak in settings?
Sometimes I spend a lot of file deleting this uneccessary duplicate versions.

**Comment by JaspalSuri** [comment]
Hi @supermensa,

I'm sorry to hear that. We haven't heard of any similar issues lately, especially since releasing v3.5 of our apps which included several significant underlying changes that helped prevent issues like this. Can you let us know what versions of the desktop and mobile apps you're currently using and what operating systems you're using them on? Are you self-hosting by any chance?

My current understanding is this: by simply typing too fast while writing inside a note, and on only one device at a time, the app generates a conflicted copy at some point. Is that correct?

**Comment by supermensa** [comment]
I don't think it is the speed of writing that causes it, but the conflicted copy is created in the middle of writing a word. Sometimes 1 or 2 copies are created in a matter of seconds.

VERSIONS
On desktop version is 3.6.0.
IOS version is 3.5.15 (41).

**Comment by supermensa** [comment]
Console output:

```
app.js:2
SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at xt.stateChangeHandlerForRequest (app.js:2)
    at XMLHttpRequest.e.onreadystatechange (app.js:2)
   
sync.standardnotes.org/items/sync:1
Failed to load resource: net::ERR_NETWORK_CHANGED

```

**Comment by JaspalSuri** [comment]
> Failed to load resource: net::ERR_NETWORK_CHANGED

Thank you for taking the time to share that! Are you using a VPN or experiencing any issues with your network by any chance? That's generally what causes the app to create conflicted notes.

**Comment by supermensa** [comment]
No, not using a VPN or experiencing network problems.
I don't know how the app is programmed, but I would think that in the event of the app failing to reach a network, it would seem beneficial to delay the conflicted copy locally.
I.e. Changes made in a matter of 1-2 minutes would not be considered critical in most use cases, in the case that the network fails.

**Comment by supermensa** [comment]
I can confirm that error persists with version 3.6.1, OS X.

**Comment by JaspalSuri** [comment]
@supermensa, thank you for letting me know. If you'd like, please email help@standardnotes.org or standardnotes@protonmail.com with your notes syncing account's username and a link to this thread so that way we can see if the server logs (for when notes were synced) can provide any useful insights or if there are any fixes that can be applied to your account.

**Comment by JaspalSuri** [comment]
Hi @supermensa, I just wanted to let you know that the dev team is currently looking into what the best solution would be for handling this issue. 

I also had another question pertaining to your network - do you use a DNS resolution service (such as OpenDNS, Google's DNS, or CloudFlare's WARP)?

**Comment by JaspalSuri** [comment]
Thank you for letting us know.

**Comment by supermensa** [comment]
Latest version 3.6.8. Console errors:
<img width="553" alt="standard-notes-sync" src="https://user-images.githubusercontent.com/1273852/117531243-f0f4a780-afe1-11eb-9fbc-7083a0269e61.png">

**Comment by JaspalSuri** [comment]
Hi @supermensa, the dev team attempted to address this issue that you are experiencing [here](https://github.com/standardnotes/snjs/pull/239), but I'm sorry to hear that it hasn't resolved the issue yet. 

Can you try [creating a new account](https://app.standardnotes.org), sign into it on the desktop/[web](https://app.standardnotes.org) and mobile apps, and let us know if the issue exists in that account so that way we know that the issue isn't account specific? Does your existing account offer the ability to upgrade the encryption of your account via the desktop app?

**Comment by supermensa** [comment]
1. Yes, I'll try that and report back.
3. No, I can't see anywhere that it does. I'm using a free account.

**Comment by JaspalSuri** [comment]
Hi @supermensa, I apologize for the delayed response as the notification for updates to this thread slipped past me. 

Thank you for taking the time to try that! The upgrade is free, so it would have appeared if your account was created several months ago. 

>It's hard to reproduce unless I sit and type for hours, but I have a suspicion that the amount of encrypted notes might also be a part? (I have close to 300 notes in the app at the moment).

I see, I thought the app was generating several conflicted copies as your were typing out the note. Thank you for providing those additional details. We'll try to reproduce the issue and keep an eye out for similar reports! 🙂

**Comment by supermensa** [comment]
No, I wasn’t able to reproduce in the 5 minutes I tried. Could be a number of things, but again: to properly reproduce I think the amount of notes would make a difference.If nothing else, I think the sync is a bit too effective in trying to sync files whilst writing a sentence. ---- On Wed, 19 May 2021 11:39:21 +0200  ***@***.******@***.***> wrote ---- 
Hi @supermensa, I apologize for misunderstanding. Were you able to reproduce the issue on a new account? I'll check in with the dev team on this again.

**Comment by JaspalSuri** [comment]
Thank you for letting me know. We'll continue investigating.

**Comment by vtwang** [comment]
Are there any updates on this issue? I get [Conflicted Copy] all the time when typing in a note, especially when I'm connected to a VPN. It happens on all editor/note types, including plain text.

VERSIONS AND PLATFORMS
OS X desktop version is 3.20.0.

**Comment by supermensa** [comment]
@vtwang Yes, same here - still happens for me. A real pain, especially mid-sentence diffs. Glad to hear I'm not the only one, though.

**Comment by moughxyz** [comment]
Please see [Microupdate: Server status and conflicts](https://blog.standardnotes.com/35571/microupdate-server-status-and-conflicts) and the subsequent [Microupdate: Resolution regarding server status](https://blog.standardnotes.com/35591/microupdate-resolution-regarding-server-status).

**Comment by vtwang** [comment]
Thank you for all the hard work and resolving the issue!
Unfortunately I seem to be having sync issues again. I started to get [Conflicted Copy] as of 2 days ago.
